README.md

```diff
@@ -71,7 +71,7 @@ we intend to provide high-quality components as part of this repository, we ackn
 for prime time. As such, each component should list its current stability level for each telemetry signal, according to
 the following definitions:
 
-### In development
+### Development
```

Comment:
This rename will break a lot of links in contrib right?

Reply:
Yes, I'll fix them once this is merged